ecs-cli/modules/commands/flags.go

```diff
@@ -48,6 +48,7 @@ const (
 	SubnetIdsFlag                   = "subnets"
 	VpcIdFlag                       = "vpc"
 	InstanceTypeFlag                = "instance-type"
+	InstanceRoleFlag                = "role"
```

Comment:
People generally prefer shorter names for command line tools, do you really have to be super explicit here when users can always go to the help text to read about what the flag is?